plugins/outputs/mqtt/mqtt.go

```diff
@@ -25,6 +25,9 @@ var sampleConfig = `
   # username = "telegraf"
   # password = "metricsmetricsmetricsmetrics"
 
+  ## client ID, if not set a random ID is generated
+  # client_id = "telegraf"
```

Comment:
please set this to `client_id = ""` in the config file, since this is the default